examples/06-verif-manual/vm-006-pinched_cylinder.py

```diff
@@ -0,0 +1,391 @@
+# Set up the element type (a shell-type).
+
+# Define the element type number.
+def define_element(mapdl, elem_type):
+    # Type of analysis: Static.
+    mapdl.antype("STATIC")
+
+    # Define element number.
+    elem_num = 1
+
+    if elem_type == "SHELL181":
+
+        # Element type: SHELL181.
+        mapdl.et(elem_num, elem_type)
+
+        # Special Features are defined by keyoptions of shell element:
+
+        # KEYOPT(3)
+        # Integration option:
+        # Full integration with incompatible modes.
+        mapdl.keyopt(elem_num, 3, 2)  # Cubic shape function
+
+    elif elem_type == "SHELL281":
+
+        # Element type: SHELL181.
+        mapdl.et(elem_num, "SHELL281")
+
+    # Show currently defined element type.
+    print(mapdl.etlist())
+    return mapdl
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material properties, where:
+# Young Modulus is :math:`E = 10.5 \cdot 10^6 psi`,
+# Poisson's ratio is :math:`Nu = 0.3125`.
+
+# Define material number.
+mat_num = 1
+
+# Define material properties.
+def define_material(mapdl):
+    # Define material properties.
+    mapdl.mp("EX", mat_num, 10.5E6)
+    mapdl.mp("NUXY", mat_num, 0.3125)
+    print(mapdl.mplist())
+    return mapdl
+
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a shell element.
+
+# Define cross-section number and thickness of the shell element.
+sec_num = 1
+t = 0.094
+
+# Define shell cross-section.
+def define_section(mapdl):
+    # Define shell cross-section.
+    mapdl.sectype(secid=sec_num, type_="SHELL", name="shell181")
+    mapdl.secdata(t, mat_num, 0, 5)
+    return mapdl
+
+
+###############################################################################
+# Define Geometry
+# ~~~~~~~~~~~~~~~
+# Set up the keypoints and create the area through the keypoints.
+
+# Define geometry of the simplified mathematical model.
+def define_geometry(mapdl):
+    global top_keypoint
+    # Define global cylindrical coordinate system.
+    mapdl.csys(1)
+
+    # Define keypoints by coordinates.
+    mapdl.k(1, 4.953)
+    mapdl.k(2, 4.953, "", 5.175)
+
+    # Generates additional keypoints from a pattern of keypoints.
+    mapdl.kgen(2, 1, 2, 1, "", 90)
+
+    # Create an area through keypoints.
+    mapdl.a(1, 2, 4, 3)
+
+    # Define the number of the keypoint where F is applied using inline function.
+    top_keypoint = mapdl.queries.kp(4.953, 90, 0)
+
+    # Plot the lines.
+    mapdl.lplot(color_lines=True, cpos='iso')
+
+    # Plot the area using PyVista properties.
+    mapdl.aplot(title="Display the selected area",
+                cpos="iso",
+                vtk=True,
+                color="#06C2AC",
+                show_line_numbering=True,
+                show_area_numbering=True,
+                show_lines=True)
+    return mapdl
```

Comment:
There's no reason to return `mapdl` in any of these functions as we are modifying it "in place". In fact, we don't even need to provide it as an input since `mapdl` is already available in the main namespace.